projects/kit/components/select/select.directive.ts

```diff
@@ -0,0 +1,54 @@
+import {computed, Directive, inject} from '@angular/core';
+import {tuiAsControl, TuiControl} from '@taiga-ui/cdk/classes';
+import {tuiValueBinding} from '@taiga-ui/cdk/utils/dom';
+import {tuiAsOptionContent} from '@taiga-ui/core/components/data-list';
+import type {TuiTextfieldAccessor} from '@taiga-ui/core/components/textfield';
+import {
+    tuiAsAuxiliary,
+    tuiAsTextfieldAccessor,
+    TuiSelectLike,
+    TuiWithTextfield,
+} from '@taiga-ui/core/components/textfield';
+import {tuiDropdownEnabled, tuiDropdownOpen} from '@taiga-ui/core/directives/dropdown';
+import {TuiItemsHandlersDirective} from '@taiga-ui/core/directives/items-handlers';
+
+import {TuiSelectOption} from './select-option/select-option.component';
+
+@Directive({
+    standalone: true,
+    selector: 'input[tuiSelect]',
+    providers: [
+        tuiAsOptionContent(TuiSelectOption),
+        tuiAsTextfieldAccessor(TuiSelectDirective),
+        tuiAsControl(TuiSelectDirective),
+        tuiAsAuxiliary(TuiSelectDirective),
+    ],
+    hostDirectives: [TuiWithTextfield, TuiSelectLike],
+    host: {
+        '[disabled]': 'disabled()',
+        '(blur)': 'onTouched()',
+        '(input)': '$event.inputType?.includes("delete") && setValue(null)',
```

Comment:
Doesn't it come from select-like?

Reply:
`TuiSelectLike` just updates native value of `<input>`. We should also update control value on backspace/delete and etc